README.md

```diff
@@ -57,13 +57,13 @@ Follow these steps to launch a Carfield simulation:
 * Compile tests for Carfield. Tests resides in `sw/tests`.
 
   ```
-  // Compile Safety Island standalone software
+  # Compile Safety Island standalone software
```

Comment:
Since we added the script sourcing in the make targets, can we remove the two additional lines here? So the unique command will be `make car-sw-build`